src/pip/_internal/cli/cmdoptions.py

```diff
@@ -495,9 +495,11 @@ def only_binary():
     '--platform',
     dest='platform',
```

Comment:
Nit: Might be a good idea to use the plural form here (same goes for `abi`).

Reply:
Slight disagreement here: The flags will still be named `--platform` and `--abi`, so it feels to me like the names of the fields in `options` should be named likewise. Note that the variables which these fields are then copied to are subsequently named `abis` and `platforms`.

Having says this, let me know if you feel strongly about this, and I'll be happy to make the change.

Reply:
I personally agree with @uranusjr.
Cf https://github.com/pypa/pip/blob/ea74bd3d9085234601a0a6199b2ad65c6b6a002f/src/pip/_internal/cli/cmdoptions.py#L395-L396
or https://github.com/pypa/pip/blob/ea74bd3d9085234601a0a6199b2ad65c6b6a002f/src/pip/_internal/cli/cmdoptions.py#L408-L409
or https://github.com/pypa/pip/blob/ea74bd3d9085234601a0a6199b2ad65c6b6a002f/src/pip/_internal/cli/cmdoptions.py#L777-L778
or https://github.com/pypa/pip/blob/ea74bd3d9085234601a0a6199b2ad65c6b6a002f/src/pip/_internal/cli/cmdoptions.py#L790-L791
or most of the options in this file with the action `append` (even if not all)

Reply:
Thanks for the pointers to prior art! Given that it's an established convention, I certainly agree - Just pushed an update.